components/Form/Form.js

```diff
@@ -20,71 +21,90 @@ export default function FormToCreateShoppingItem({
-            />
+          <StyledLabel htmlFor="name">
+            {/* <Image
+              src={`/icons/asterisk-svgrepo-com.svg`}
+              alt={asterisk}
+              // style={{ objectFit: "contain" }}
+              // width={140}
+              // height={140}
+              /> */}
+            {item.id ? "item to be edited*: " : "new item*:"}
           </StyledLabel>
+          <StyledInput
+            id="name"
+            name="name"
+            type="text"
+            required
+            placeholder="e.g., salt"
+            defaultValue={item.name || ""}
+          />
+          <StyledLabel htmlFor="quantity">quantity*:</StyledLabel>
+          <StyledInput
+            id="quantity"
+            name="quantity"
+            type="number"
+            required
+            defaultValue={item.quantity || ""}
+          />
+          <StyledLabel htmlFor="category">category*:</StyledLabel>
+          <select
+            id="category"
+            name="category"
+            // data-js="category"
+            required
+            defaultValue={
+              // I have replaced the line below with "item.id" because it didn't work anymore after the suggested refactoring;
+              // probably because I had undone handing down the categories(?)
+              // mode === "edit"
+              // item.id
+              //   ?
+              item.category || ""
+              // :
+              // replace "" by "default" or undo "" > "default" elsewhere
+              // ""
+            }
+          >
+            <option value="">please select a category</option>
+            {categories.map((category) => (
+              <option key={category.name} value={category.name}>
+                {category.name}
+              </option>
+            ))}
+            ;
+          </select>
 
-          <StyledLabel>
-            category*:
-            <select
-              key="category"
-              name="category"
-              data-js="category"
-              required
-              defaultValue={mode === "edit" ? item.category : ""}
-            >
-              <option value="">please select a category</option>
-              {categories.map((category) => (
-                <option key={category.name} value={category.name}>
-                  {category.name}
-                </option>
-              ))}
-              ;
-            </select>
-          </StyledLabel>
-          <StyledLabel>
-            comment:
-            <StyledInput
-              name="comment"
-              type="text"
-              defaultValue={item.comment || ""}
-              placeholder="Enter comments here..."
-            />
-          </StyledLabel>
+          <StyledLabel htmlFor="comment">comment:</StyledLabel>
+          <StyledInput
+            name="comment"
+            type="text"
+            defaultValue={item.comment || ""}
+            placeholder="preferably sea salt"
+          />
```

Comment:
You are missing an `id` here to connect the input field with the corresponding label.

```suggestion
          <StyledInput
            id="comment"
            name="comment"
            type="text"
            defaultValue={item.comment || ""}
            placeholder="preferably sea salt"
          />
```

Reply:
Done.